www/js/control/SettingRow.tsx

```diff
@@ -12,18 +12,24 @@ type Props = {
   descStyle?: any;
 };
 const SettingRow = ({ textKey, iconName, action, desc, switchValue, descStyle }: Props) => {
-  const { t } = useTranslation(); //this accesses the translations
-  const { colors } = useTheme(); // use this to get the theme colors instead of hardcoded #hex colors
+  const { t } = useTranslation(); // Access translations
+  const { colors } = useTheme(); // Get theme colors
 
   let rightComponent;
   if (iconName) {
-    rightComponent = <List.Icon icon={iconName} />;
+    rightComponent = <List.Icon icon={iconName} aria-hidden={true} />;
   } else {
-    rightComponent = <Switch value={switchValue} />;
+    rightComponent = (
+      <Switch
+        value={switchValue}
+        accessibilityLabel={t(textKey as any)}
+        accessibilityHint={switchValue ? t('Currently enabled') : t('Currently disabled')}
+      />
+    );
   }
   let descriptionText;
   if (desc) {
-    descriptionText = { desc };
+    descriptionText = desc;
   } else {
```

Comment:
I don't understand the reason for this

Reply:
Resolved in https://github.com/e-mission/e-mission-phone/pull/1118/commits/b0596e0741eee36a946b68115468b62aef7671c8